releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -255,6 +244,15 @@ services:
       edgex-network:
         aliases:
             - edgex-proxy
+    environment:
+      <<: *common-variables
+      KongURL_Server: kong
+      SecretService_Server: edgex-vault
+      SecretService_SecretService_TokenPath: /tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json
+      CACertPath: /tmp/edgex/secrets/ca/ca.pem
+      SecretService_SNIS: "edgex-kong"
+      Clients_RulesEngine_Host: edgex-support-rulesengine
```

Comment:
```suggestion
```

edgex-proxy doesn't have a dependency on either rules engine or or device-virtual.  These can be removed entirely.